src/app/pages/ParatimeDashboardPage/TransactionsChartCard.tsx

```diff
@@ -15,12 +16,20 @@ import { PercentageGain } from '../../components/PercentageGain'
 import startOfHour from 'date-fns/startOfHour'
 import { useRequiredScopeParam } from '../../hooks/useScopeParam'
 
+const getLabels = (t: TFunction): Record<ChartDuration, string> => ({
+  [ChartDuration.TODAY]: t('chartDuration.lastHour'),
+  [ChartDuration.WEEK]: t('chartDuration.lastDay'),
+  [ChartDuration.MONTH]: t('chartDuration.lastDay'),
+  [ChartDuration.ALL_TIME]: t('chartDuration.lastDay'),
+})
+
```

Comment:
Why do we have this mapping repeated by card?

Reply:
We want to have different label for each chart.

Reply:
Ahh I see, the very last item is different. (I would probably still prefer to have a common function with a parameter, but whatever.)